ansible/inventories/devnet-0/group_vars/besu.yaml

```
@@ -28,5 +28,6 @@ besu_container_command_extra_args:
   - --rpc-http-api=ADMIN,DEBUG,ETH,MINER,NET,TRACE,TXPOOL,WEB3
   - --sync-mode=FULL
   - --data-storage-format=BONSAI
+  - --Xbonsai-limit-trie-logs-enabled=false
```

```suggestion
  - --bonsai-limit-trie-logs-enabled=false
```

I've actually just merged a PR to remove the X so this way is more future-proof (although Xbonsai... will still work it's just deprecated).

Only caveat is this removal of the X may not make the cut for the upcoming release, so if using the yet to be released 24.6.0, use the X, otherwise no X is better.